src/globals.h

```diff
@@ -33,6 +33,8 @@ extern struct globals {
 	bool sigcheck;
 	bool timecheck;
 	bool wait_for_scripts;
+	bool is_interactive;
```

Comment:
Instead of 2 boolens, can't we have an enum with values interactive, assume_yes, assume_no? So we won't end up in cases where is_interactive == false and assume_yes == true.

Reply:
good idea, I'll make the change